tests/Backend/Snowflake/TableExportTest.php

```diff
@@ -42,29 +42,19 @@ public function testSyncExportShouldReturnErrorForLargeNumberOfCols()
     public function testSyncExportMax30cols()
     {
         $cols = [];
-        $cols[] = implode(',', array_map(function ($colNum) {
-            return "col_{$colNum}";
-        }, range(1, 30)));
-
-        $cols[] = implode(',', array_map(function ($colNum) {
-            return "data_{$colNum}";
-        }, range(1, 30)));
-
-        $cols[] = implode(',', array_map(function ($colNum) {
-            return "data_{$colNum}";
-        }, range(1, 30)));
-
-        $cols[] = implode(',', array_map(function ($colNum) {
-            return "data_{$colNum}";
-        }, range(1, 30)));
-
+        $rows = 4;
+        for ($i = 0; $i < $rows; $i++) {
```

Comment:
Tohle tady asi nemá co dělat. Když tak to vyhodím. Ale šel jsem kolem a nějak se mi nezdálo to volání

Reply:
nechme to klidně tady..